google-map-directions.html

```diff
@@ -83,6 +91,24 @@
       travelMode: 'DRIVING',
 
       /**
+       * Options for rendering.
+       *
+       * @attribute rendererOptions
+       * @type object
+       * @default {}
+       */
+      rendererOptions: {},
+
+      /**
+       * A panel for displaying directions
+       *
+       * @attribute panel
+       * @type HTMLElement
+       * @default null
+       */
+      panel: null,
```

Comment:
I'm not sure we need an attribute for this. Can't someone just set the renderOptions?